### DISCUSSION TITLE
Mappen

### DISCUSSION THREAD

**Discussion by mohbusaidy** [neuefische/questions#20] [Web] [answered]
Hey ich hab eine frage zum mappen, ich versuche objekte die in arrays liegen, mit einer map function in meine produkt seite dar zu stellen. Das klappt jetzt auch! Aber sobald ich ein zweites objekt in meine data.js hinzufüge war das ziel das sich eine zweite karte in meiner Produkt seite generiert. Da, hänge ich leider mein nächstes objekt taucht in meiner karte die schon vorhanden ist mit auf. Ich hoffe ich hab meine problemstellung gut genug erklärt 😅

**Comment by actyralf** [comment]
Kannst Du ein Code-Snippet Deiner `map` Funktion anfügen? Danke!

**Comment by mohbusaidy** [comment]
<img width="200" alt="Bildschirmfoto 2022-07-27 um 09 36 06" src="https://user-images.githubusercontent.com/104504053/181189
<img width="200" alt="Bildschirmfoto 2022-07-27 um 09 36 24" src="https://user-images.githubusercontent.com/104504053/181189661-dbf12ad8-2628-412f-b8e9-e14238d35916.png">
494-dff0152a-68d9-4f3b-adac-00008f96c8ea.png">
<img width="200" alt="Bildschirmfoto 2022-07-27 um 09 35 51" src="https://user-images.githubusercontent.com/104504053/181189721-3520ca5b-2ca5-449f-a30a-dea71b6b32ce.png">
<img width="200" alt="Bildschirmfoto 2022-07-27 um 09 35 34" src="https://user-images.githubusercontent.com/104504053/181189751-2c8da730-acc1-47da-916a-15b535b53f70.png">

**Comment by actyralf** [comment, marked as answer]
The problem is that you use the `card` CSS class for your `<main>` tag. This leads to one big card (main) containing all mapped products.
Just remove it there and use it in your `Product` component instead.
